docs/core/config/auth/databases/ldap.md

```diff
@@ -251,8 +251,7 @@ If you need to connect to ldaps secured against a custom certificate of
 authority (CA), you will need to install the custom CA on your system.
 
 For OpenLDAP, by default, the CA must be installed under the directory
-specified in the `TLS_CACERTDIR` option found under `/etc/openldap/ldap.conf`
-(default value is `/etc/openldap/certs`).
+specified in the `TLS_CACERTDIR` option (default value is `/etc/openldap/certs`).
```

Comment:
why remove the config file path? This is openldap's config file, not dovecot.

Reply:
oops